integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/AbstractIT.java

```diff
@@ -259,4 +274,18 @@ protected String readGitCommitIdFromGitFile() {
       return "";
     }
   }
+
+  private static boolean isGravitinoServerUp() {
+    try {
+      URL url = new URL("http://localhost:8090");
+      HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+      connection.setRequestMethod("GET");
+      connection.setConnectTimeout(1000);
+      connection.connect();
+      int responseCode = connection.getResponseCode();
+      return responseCode == 200;
```

Comment:
You need to disconnect the connection in case of resource leaks and please refer to:

https://github.com/datastrato/gravitino/blob/2c41f4dbd876342335475009ed546d8e06a94945/integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/MiniGravitino.java#L249-L256

Reply:
thank you. I will modify according this link.